src/main/java/com/kodilla/ecommercee/domain/User.java

```diff
@@ -0,0 +1,24 @@
+package com.kodilla.ecommercee.domain;
+
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+
+@AllArgsConstructor
+@NoArgsConstructor
+@Getter
+@Entity
+public class User {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.AUTO)
```

Comment:
samo @GeneratedValue bez podania agumentów w nawiasie wystarcza, domyślnie generuje liczby.

Reply:
Tak to wszystko ok .

Reply:
poprawione